grass-gis-addons/m.analyse.buildings/r.extract.buildings/r.extract.buildings.py

```diff
@@ -71,36 +66,37 @@
 # %option
 # % key: max_fd
 # % type: double
-# % required: no
+# % required: yes
 # % multiple: no
 # % label: Maximum value of fractal dimension of identified objects (see v.to.db)
 # % answer: 2.1
 # % guisection: Parameters
 # %end
 
 # %option
-# % key: ndvi_thresh
+# % key: ndvi_perc
 # % type: integer
 # % required: no
 # % multiple: no
-# % label: Define fix NDVI threshold (on a scale from 0-255) instead of estimating it from FNK
+# % label: NDVI percentile in vegetated areas to use for thresholding
+# TODO: make radio button and add default
```

Comment:
radio button missing? (and afterwards remove todo)

Reply:
I would do this in a separate MR for various addons.